actinia-alpine/Dockerfile

```diff
@@ -27,16 +27,23 @@ USER root
 COPY --from=version /actinia-docker-version.txt /
 
 # # ESA SNAP SETUP
-# ENV LD_LIBRARY_PATH ".:$LD_LIBRARY_PATH"
-# COPY --from=snap /root/.snap /root/.snap
-# COPY --from=snap /usr/local/snap /usr/local/snap
+ENV LD_LIBRARY_PATH ".:$LD_LIBRARY_PATH"
+COPY --from=snap /root/.snap /root/.snap
+COPY --from=snap /usr/local/snap /usr/local/snap
 # RUN (cd /root/.snap/snap-python/snappy && pip3 install .)
 # RUN /usr/bin/python3 -c 'from snappy import ProductIO'
 # RUN /usr/bin/python3 /root/.snap/about.py
+# add gpt to PATH
+ENV PATH="${PATH}:/usr/local/snap/bin"
+# test gpt
+RUN gpt -h
 
 # GRASS GIS SETUP
 COPY --from=grass /usr/local/bin/grass /usr/local/bin/grass
 COPY --from=grass /usr/local/grass* /usr/local/grass/
+# COPY --from=grass /usr/lib/gdalplugins/*_GRASS* /usr/lib/gdalplugins/
```

Comment:
If you don't copy the GDAL plugins, you might loose quite a lot of GDAL/OGR drivers, depending on the configuration of GDAL.

Reply:
Note that this commented line is just added in this PR, nothing was copied before either.

The missing GDAL plugins that are on the `osgeo/grass-gis:releasebranch_8_3-alpine` but not on the base image `actinia:alpine-dependencies` are:

```
- gdal_PNG.so
- gdal_GRASS.so
- ogr_GMLAS.so
- ogr_GRASS.so
- ogr_LIBKML.so
- ogr_MSSQLSpatial.so
- ogr_ODBC.so
```

but `gdal_GRASS.so` and `ogr_GRASS.so` are installed [here](https://github.com/actinia-org/actinia-docker/pull/70/files#diff-cc74acbbcf43b6df712226647ffbff285c3a5a47cb523962d509221a78ab5ae9R136)

Reply:
OK, so this Docker file uses GDAL from `actinia:alpine-dependencies`. In this case it would cause conflicts to copy GDAL plugins from `osgeo/grass-gis:releasebranch_8_3-alpine`. But GRASS has been compiled against GDAL in `osgeo/grass-gis:releasebranch_8_3-alpine`, therefore the GDAL versions in the two different images must be the same, otherwise you get errors running GRASS. Considering this, your solution to compile the GDAL GRASS plugin later on in this Dockerfile seems to be the safest option.

Reply:
The GDAL versions in the grass image and actinia image are the same `GDAL 3.6.4, released 2023/04/17` but this is definitely something we need to be aware of!